frontend/src/metabase/components/Calendar/Calendar.module.css

```diff
@@ -1,32 +1,32 @@
-.Calendar-weeks {
+:global .Calendar-weeks {
```

Comment:
I'm not super familiar with [scoping without parentheses](https://github.com/webpack-contrib/css-loader?tab=readme-ov-file#scope). But could it be possible to wrap these within parentheses for consistency?